documentation/examples/custom-sd/adapter/adapter.go

```diff
@@ -34,6 +36,31 @@ type customSD struct {
 	Labels  map[string]string `json:"labels"`
 }
 
+//return targetsHash.Sum64() ^ labelsHash.Sum64(), nil
```

Comment:
If you want to keep these commented lines, comments should be in the form of a full sentence. `hash turns both the targets and labels for sd into hashes, and then returns the targetHash ^ labelsHash.` as an example.